src/operator/main.go

```diff
@@ -161,10 +158,17 @@ func main() {
 		logrus.WithError(err).Fatal("unable to init index for ingress")
 	}
 
+	otterizeCloudClient, ok, err = otterizecloud.NewClient(context.Background())
+	if !ok {
+		logrus.Info("missing configuration for cloud integration, disabling cloud communication")
+	} else if err != nil {
```

Comment:
Check the error even if !ok, since when an error is returned ok is still false. Basically, unchain the if,else

Reply:
fixed